app/bloom/templates/cronjob.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: bloom
+spec:
+  schedule: "*/15 * * * *"
+  jobTemplate:
```

Comment:
Let's add some [history limits](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#jobs-history-limits) to both successful and failed runs !